extensions/objects/protractor.js

```diff
@@ -30,11 +30,24 @@ var AlgebraicExpressionEditor = function(elem) {
       await waitFor.elementToBeClickable(
         elem, '"Algebraic Input" editor takes too long to be clickable');
       await elem.click();
-      var algebraicInputElem = element(by.css('.guppy_active'));
+      var algebraicInputElem = element.all(by.css(
+        '.protractor-test-guppy-div')).last();
       var present = await algebraicInputElem.isPresent();
       if (present) {
         await algebraicInputElem.sendKeys(text);
       }
+    },
+    getValue: async function() {
+      var algebraicInputElem = element.all(by.css(
+        '.protractor-test-guppy-div')).last();
+      var present = await algebraicInputElem.isPresent();
+      if (present) {
+        var contentElem = element.all(by.tagName('annotation')).first();
```

Comment:
Can we use a `protractor-test-*` tag here too?

Reply:
Ah, I just realized that we can use `algebraicInputElem` element to query this. And since there will only be one `annotation` tag in there, we don't need to fetch the first one explicitly.